.java-version

```diff
@@ -1 +1 @@
-21.0
+22
```

Comment:
Does Docker image pull java based .java-version?

Reply:
yes, it's using .java-version to determine JDK version to install